tests/nonhumans_test.py

```diff
@@ -0,0 +1,29 @@
+"""Test cases for nonhuman related features."""
+
+# Third-Party Libraries
+import pytest
+
+
+class TestNonhumans:
+    """Test case for nonhuman related views."""
+
+    def test_nonhumans_view(self, client):
+        """Test the nonhumans view."""
+        resp = client.get("/api/nonhumans/")
+        assert resp.status_code == 200
+
+        self.check_nonhuman_properties(resp.json)
+
+    @staticmethod
+    def check_nonhuman_properties(nonhuman):
+        """Check nonhuman object for expected properties."""
+        try:
+            assert isinstance(nonhuman[0], str)
+        except KeyError:
+            pytest.fail("expected a string")
+
+        try:
+            assert isinstance(nonhuman, list)
+            assert len(nonhuman) >= 3
+        except KeyError:
```

Comment:
KeyError is only raised if a dict fails to find a key when you use a direct lookup

Reply:
Good point! Resolved with e3d9595830337df26d5c91a6890d6364d5eed8fe